SetFlags.cmake

```diff
@@ -86,7 +86,11 @@ macro(set_flags)
 		else()
 			add_flags_cxx("-pthread")
 		endif()
-
+	elseif(ANDROID)
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
You won’t be dealing with it, considering this will be in a separate repository. It’ll probably end up being me dealing with it :P